### ISSUE TITLE
Scanning broken for Quake

### ISSUE THREAD

**Issue by metchebe** [libretro/libretro-database#1295]
As commented in the relevant pull request #1239, the scanner does not work with the current .rdb database.

I can confirm that reverting to the previous rdb from release [v1.9.10](https://github.com/libretro/libretro-database/releases/tag/v1.9.10) makes the scanner funcional again.

I understand the PR was to fix some other error but it seems to be incompatible with the scanner.

**Comment by CygnusHyoga** [comment]
Hello, I need help to create Quake retroarch playlist. I'm currently with retroarch for OSX, the games work perfectly, both the base game and the expansions.

The problem I have is that when I scan the directory where I have the quakes, only the *.pak appear but it does not match the libretro database game name. 

Libretro Quake Database: https://db.libretro.com/Quake/index-0.html
Quake Dat Libretro: https://github.com/libretro/libretro-database/blob/master/dat/Quake.dat

Example how I see the Playlist:
[https://www.reddit.com/r/RetroArch/comments/vx91af/quake_playlists_help/](url)

I leave the directory structure and the crcs, which I currently have.

Directories Structure
```
ls -R /Volumes/emu/roms/Quake
dope
hypnotic
id1
rogue

/Volumes/emu/roms/Quake/dopa:
pak0.pak

/Volumes/emu/roms/Quake/hypnotic:
music pak0.pak

/Volumes/emu/roms/Quake/hypnotic/music:
track02.ogg track04.ogg track06.ogg track08.ogg
track03.ogg track05.ogg track07.ogg track09.ogg

/Volumes/emu/roms/Quake/id1:
music pak0.pak pak1.pak

/Volumes/emu/roms/Quake/id1/music:
track02.ogg track04.ogg track06.ogg track08.ogg track10.ogg
track03.ogg track05.ogg track07.ogg track09.ogg track11.ogg

/Volumes/emu/roms/Quake/rogue:
music pak0.pak

/Volumes/emu/roms/Quake/rogue/music:
track02.ogg track04.ogg track06.ogg track08.ogg
track03.ogg track05.ogg track07.ogg track09.ogg
```

Files CRCs
```
74e3b084 /Volumes/emu/roms/Quake/id1/pak0.pak
2e803216 /Volumes/emu/roms/Quake/id1/pak1.pak
38b70ce8 /Volumes/emu/roms/Quake/rogue/pak0.pak
d17835fb /Volumes/emu/roms/Quake/hypnotic/pak0.pak
8a68d267 /Volumes/emu/roms/Quake/dopa/pak0.pak
```

**Comment by RobLoach** [comment]
If you run an automated scan, it should work. All of those CRCs are represented in the DAT.

**Comment by CygnusHyoga** [comment]
Yes, with automated scan works perfect! I always do it with manual scan, but I confirm with automated scan works OK. Thanks for the help!
